src/styles/cardStyles.scss

```diff
@@ -0,0 +1,88 @@
+@import "./cardBlock/card.scss";
+@import "./cardBlock/stars.scss";
+@import "./value.scss";
+
+.text-Main {
+  transition-duration: $duration;
+  color: $Main-accent;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 40px 0 0;
+}
+
+.text-Secondary {
+  color: $Secondary;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  margin: 4px 0 0;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 24px;
+  line-height: 18px;
+
+  &__text {
+    color: $Secondary;
+    font-size: 12px;
+    font-weight: 400;
+  }
+
+  &__value {
+    color: $Main-accent;
+    text-align: right;
+    font-size: 16px;
+    font-weight: 700;
+  }
+}
+
+.rating {
+  display: flex;
+  margin-top: 16px;
+  width: 100%;
+  height: 16px;
+  justify-content: space-between;
+
+  &__text {
+    color: $Main-accent;
+    text-align: right;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    align-self: flex-end;
+  }
+}
+
+.button {
```

Comment:
all of the selectors in this file should be elements of the card block

theoretically, it could be separate blocks, but in this case, according to BEM rules, each one should be in a separate file. it sounds too much, isn't it? So you grouped it in one file with `cardStyles` name, which means all of these styles are related to the card. So that's the answer - if it's true they should be elements of the card. So move them and fix names in HTML accordingly